.eslintrc.js

```diff
@@ -32,5 +32,8 @@ module.exports = {
         node: true
       }
     }
-  ]
-};
+  ],
+  globals: {
+    "Promise": true
+  }
+};
```

Comment:
Why is this added? We should avoid global references when possible.

Reply:
Add a newline after the last line of the file. If you do that, `.eslintrc.js` will no longer show up in this PR and things will be a bit cleaner.

Reply:
Strange -- thought I did these changes from the first PR -- maybe I rewrote over them when there was a conflict. Either way,  taken these changes!